ruby-challenges.rb

```diff
@@ -10,21 +10,66 @@
 letter_t = 't'
 # Expected output: ['tea', 'water', 'soda water']
 
+def findWords(arr, ltr)
+    arr.filter {|str| str.include? ltr }
+end
+
+p findWords(beverages_array, letter_o)
+p findWords(beverages_array, letter_t)
+
 
 # -------------------2) Create a method that takes in a hash and returns one array with all the hash values at their own index and in alphabetical order. No nested arrays. Use the test variable provided.
 # HINT: Google 'ruby get rid of nested arrays'
 
 us_states = { northwest: ['Washington', 'Oregon', 'Idaho'], southwest: ['California', 'Arizona', 'Nevada'], notheast: ['Maine', 'New Hampshire', 'Vermont'] }
 # Expected output: ['Arizona', 'California', 'Idaho', 'Maine', 'Nevada', 'New Hampshire', 'Oregon', 'Vermont', 'Washington'] 
 
+def removeNestedArrays(hash)
+    array = []
+    hash.values.each do |value|
+        value.map do |element| 
+            array << element 
+        end
+    end
+    array.sort
+end
```

Comment:
This looks great!  There is also a flatten method that will do some of the work for you.